src/ansys/aedt/toolkits/common/utils.py

```diff
@@ -146,7 +146,7 @@ def check_backend_communication(url_call):
         response = requests.get(url_call + "/health")
         return response.ok
     except requests.exceptions.RequestException:
-        logger.error("Failed to check backend communication.")
+        print("Failed to check backend communication.")
```

Comment:
Why a print instead of a log message? I am sure you have a good reason :)

Reply:
logger is not defined at this level, so it was crashing the code. In another PR, we can check if it is worth adding the global logger.